src/backtick/core.clj

```diff
@@ -8,10 +8,16 @@
   (:gen-class))
 
 (defn register [name f]
-  (swap! engine/workers assoc name f))
+  ;; We store both the name => fn mapping (needed by submit-worker)
+  ;; and fn => name mapping (needed by schedule)
+  (swap! engine/workers assoc
+         name f
+         f name))
```

Comment:
Do I have to pass `name` as a string? What if I could just call it like this?
```
(defn foobar [] ...)
(register foobar)
```
Seems like you could use the function metadata to construct the string name, e.g. `(-> foobar var meta :name)`?